spawn-host.patch

```diff
@@ -0,0 +1,21 @@
+--- ~/src_assets/linux/assets/apps.json
++++ ~/src_assets/linux/assets/apps.json
+@@ -12,15 +12,15 @@
+       "image-path": "desktop.png",
+       "prep-cmd": [
+         {
+-          "do": "xrandr --output HDMI-1 --mode 1920x1080",
+-          "undo": "xrandr --output HDMI-1 --mode 1920x1200"
++          "do": "flatpak-spawn --host xrandr --output HDMI-1 --mode 1920x1080",
++          "undo": "flatpak-spawn --host xrandr --output HDMI-1 --mode 1920x1200"
+         }
+       ]
+     },
+     {
+       "name": "Steam Big Picture",
+       "detached": [
+-        "setsid steam steam://open/bigpicture"
++        "flatpak-spawn --host setsid steam steam://open/bigpicture"
```

Comment:
so this require something on the host? What's the failure mode?

Reply:
In upstream we patch it with `SED`, which allows us to add/remove apps without worrying about updating a patch file.

https://github.com/LizardByte/Sunshine/blob/master/packaging/linux/flatpak/dev.lizardbyte.sunshine.yml#L233-L242

I understand `SED` can be confusing if you don't understand regex.

I don't use flatpak, but don't the `do/undo` commands also need to be patched? https://github.com/LizardByte/Sunshine/blob/master/src_assets/linux/assets/apps.json#L15-L16

> What's the failure mode?
Steam won't launch.

Reply:
sed as a failure mode that is not a failure. So things silently fail and then use complain things are broken.

Maybe you like that kind of failure mode.

Reply:
as for steam, what if the user has it as a flatpak?

Reply:
> as for steam, what if the user has it as a flatpak?

Then they can manually change it in our config UI (through web browser).